### ISSUE TITLE
CC-Aligner support for auto matching of subtitles

### ISSUE THREAD

**Issue by gabriellluz** [SubtitleEdit/subtitleedit#3332]
Hi there.

I'd like to suggest this project called CC-Aligner for auto aligning/syncing a subtitle with a video by using speech recognition.
[CC-Aligner](https://github.com/saurabhshri/CCAligner)

Basically it should allow you two things:
- getting a DVD release subtitle and sync it to a Bluray release;
- getting a SRT without timing (text to SRT, like SubtitleEdit does) and getting CC-Aligner to make timing for you.

It's just great.
You need Python37 (because of Tensorflow) in order for it to work. Doesn't work with Python2.

**Note do developers:**
Basically, in SubtitleEdit, you should open a subtitle file or a TXT file, then you have to open the video because the audio in WAV format must be extracted. Then, you run cc-aligner (like a hidden command line) on that wav file and automatically get an almost perfect timed SRT file.
